code/modules/mob/living/simple_animal/guardian/guardian.dm

```diff
@@ -542,14 +542,12 @@
 /mob/living/simple_animal/hostile/guardian/bomb/AltClickOn(atom/movable/A)
 	if(!istype(A))
 		return
-	if(src.loc == summoner)
-		src << "<span class='danger'><B>You must be manifested to create bombs!</span></B>"
-		return
 	if(istype(A, /obj/))
 		if(bomb_cooldown <= world.time && !stat)
 			var/obj/item/weapon/guardian_bomb/B = new /obj/item/weapon/guardian_bomb(get_turf(A))
+			PoolOrNew(/obj/effect/overlay/temp/guardian/phase, get_turf(A))
```

Comment:
http://puu.sh/n5itG/a142470c87.dmi
Drop it in effects.dmi
Make a new temporary overlay in overlays.dm subtype of guardian, duration 4, using this icon.
I'd suggest making it match the object's pixel x/y like swarmer overlays do.